src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+const isSorted = function(arr, compareFunction) {
+  for (let i = 1; i < arr.length; i++) {
+    if (compareFunction(arr[i - 1], arr[i]) > 0) {
+      return false;
+    }
+  }
+
+  return true;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = function(a, b) {
+    if (b.toString() === a.toString()) {
+      return 0;
+    };
+
+    return a.toString() > b.toString()
+      ? 1
+      : -1;
+  }) {
+    let swap = 0;
+
+    while (!isSorted(this, compareFunction)) {
```

Comment:
Do you really need to use `while` here?
You used `bubble sorting` algorithm, so it's enough to use two nested cycles. There is no need to check after your cycles had done if your list is sorted. It should be definitely sorted already.